src/timeline/components/stateChronicles/TimelineChronicleLegend.vue

```diff
@@ -1,7 +1,21 @@
 <template>
 <ul :style="legendStyle" v-if="inBoundsEpisodes.length > 0">
     <div v-if="endPoints">
-        <div>
+        <div @mouseover="showOptionBar = true" @mouseleave="showOptionBar = false">
```

Comment:
Could we use css to show and hide the options bar?

Reply:
@deeptailor We could, but because the style tags aren't working in the Vue components. This was my solution until then. I prefer to keep things scoped into their components as I do know implementing css in the index.html would work.

Reply:
I would also still need this logic to keep track of mouse events to switch between the defined classes to trigger the respective hide and show class in the options bar.